### DISCUSSION TITLE
How to get the relation object when the relation is user

### DISCUSSION THREAD

**Discussion by hakon-metzum** [pocketbase/pocketbase#134] [Q&A]
I do have some issues to get a user relation using expand=owner (where owner is a user reference). I get a 400 Bad Request when adding this parameter to the API request. I have tried opening up the user API with both empty rule string, and also @request.user.id != null, but I still get the error.

What is the secret sauce to get the object when the relation is a user (this is from API logged in as a user, not admin)

**Comment by ganigeorgiev** [comment]
`Relation` and `User` fields are 2 different things. The first one links Collection Records, the latter - users.
`expand` can be used only with relation fields.

I don't fully understand what actually you are trying to achieve. You cannot return the user data (email, verified, etc.) in a record request via expand.

Users have `profiles` collection attached to them, which is used for defining custom fields. If you want to use expand to return the user custom fields, you could add a `relation` field to the `profiles` collection (each user profile has `userId` field).

**Reply by hakon-metzum** [reply]
Ok, let me explain. I have a setup with projects and tasks (a project has 0 to MANY tasks). The task has an owner (a user). I would like to display the name of the owner on the task when listed. Thats why I have set the a relation to a user. I know the type "relation" is not the same as the "user" in a PocketBase sense, but how can I show the name of the user that owns a task? 

I am missing anything regarding PocketPase data models?

**Reply by ganigeorgiev** [reply]
I understand. The `profiles` collection could be used exactly for this case.

**Comment by hakon-metzum** [comment]
I do see that there is a relation called "profile". I did not see it before, and I guess this will do exactly as I need. Sorry for bugging you, and thanks for a beautiful project, and all the help

**Reply by ganigeorgiev** [reply]
Also note, that the `profiles` collection has its own rules that you may need to configure (could be edited in "Users > Edit profiles collection" cogwheel).

**Comment by hakon-metzum** [comment]
I would like to add a few things to the documentation down the road, if I may? That could help others solving things like this

**Reply by ganigeorgiev** [reply]
The documentation is part of the static landing site (built with SvelteKit).
I didn't bother open source it because it is kinda messy and a little complicated to be edited by users not familiar with the codebase, but I'll try to find some time in the future and may publish it.

In the meantime, feel free to post your suggestions here (as new discussion or in this one) and I'll review them.

**Reply by hakon-metzum** [reply]
Sure, I will. I was thinking more on the readme.md file for now, but I can write suggestions here
